src/locale/translations/en.json

```diff
@@ -546,28 +546,28 @@
     },
     "ScanAPlace": {
       "Title": "Add a visit: Mooseville only",
-      "Body": "Get notified if someone from today tests positive later.\n\nThe app saves your visits on your phone, including the dates and times you were there.\n\n**Your visits are not shared** with anyone.",
-      "Banner": "Right now, posters with QR codes are only available in Mooseville as part of a small-scale pilot project.",
+      "Body": "Get notified if public health finds an exposure risk from your visit.\n\nThe app saves your visits on your phone, including the dates and times you were there.\n\n**Your visits are not shared** with anyone.",
+      "Banner": "Right now, posters with QR codes are **only available in Mooseville** as part of a small-scale pilot project.",
       "CTA": "Next",
       "CTA2": "About adding a visit"
     },
     "ProximityExposure": "Get tested",
     "OutbreakExposed": {
-      "Title": "Possible exposure at a visit",
-      "Body": "Someone tested positive from one of your visits.",
-      "DateDesc": "Date of exposure: ",
+      "Title": "You’ve been exposed",
```

Comment:
Isn't this supposed to be: "You've been exposed at a visit"?
https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=19406%3A74597

Reply:
Since we're not adding "from close contact" for the proximity one (@timarney flagged this), we'll treat this heading the same way for the beta and not add "at a visit". https://www.figma.com/file/0pLawavG1fnBTXSYn4pdKZ/Exposure-notification?node-id=19556%3A75360

Reply:
Okay sounds good @EliseKa.